src/forms-opportunities/guidance-handlers/guidance-high-form-views-low-conversions.js

```diff
@@ -17,7 +17,7 @@ export default async function handler(message, context) {
   const { log, dataAccess } = context;
   const { Opportunity } = dataAccess;
   const { auditId, siteId, data } = message;
-  const { url, guidance, formsource } = data;
+  const { url, guidance, form_source: formsource } = data;
```

Comment:
It's very strange to use snake case and camel case both in the same data `dataAccess` vs `form_source`